spec/features/user/user_views_past_orders_spec.rb

```diff
@@ -27,15 +27,15 @@
     click_on "Checkout"
   end
 
-  scenario 'will be displayed' do
+  xscenario 'will be displayed' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.